protosocket-connection/src/connection.rs

```diff
@@ -14,6 +14,8 @@ use crate::{
     would_block, Deserializer, Serializer,
 };
 
+const BUFFER_INCREMENT: usize = 16 * (2 << 10);
```

Comment:
Does the compiler know this and only initializes it once? Thats one confusion I had and hence moved it out

Reply:
Yes. https://doc.rust-lang.org/std/keyword.const.html Also, even if it didn't, a compile-time constant value declared on the stack is near enough to free that it may as well be free. And a compile-time constant value will be trivially stripped by llvm optimization passes typical to --release builds. If you put this back and made it a `let` instead to make it per-stack-frame, I bet you the post-optimization assembly would be the same as with the local const.